straxen/plugins/daqreader.py

```diff
@@ -89,7 +89,15 @@ class DAQReader(strax.Plugin):
     data_kind = immutabledict(zip(provides, provides))
     depends_on = tuple()
     parallel = 'process'
-    rechunk_on_save = False
+    rechunk_on_save = immutabledict(
+        raw_records=False,
+        raw_records_he=False,
+        raw_records_aqmon=True,
```

Comment:
So each rechunked chunk will have max size of 200 MB, so as you're mentioned e.g. all rr_aqmon can fit into a single chunk?

Reply:
yeah for sure, the aqmon records are just a few MB even in the largest runs